include/dnssnoop.h

```diff
@@ -42,25 +50,39 @@ struct dnshdr {
 };
 
 struct inflight_dns_query {
+  // userspace tid, kernel pid
+  pid_t tid;
+  // userspace pid, kernel tgid
   pid_t pid;
-  pid_t tgid;
+  uid_t uid;
+  gid_t gid;
   u64 cgroup_id;
   u64 start_time;
-  u16 id;
-  char comm[COMM_BUF_SIZE];
-  char name[NAME_BUF_SIZE];
-  char cgroup[CGROUP_BUF_SIZE];
+  u16 transaction_id;
+  char command[COMMAND_BUF_SIZE];
+  char thread_name[THREAD_NAME_BUF_SIZE];
+  char domain_name[DOMAIN_NAME_BUF_SIZE];
+  char cgroup[CGROUP_NAME_BUF_SIZE];
 } __attribute__((packed));
 
 struct dns_query {
+  // userspace tid, kernel pid
+  pid_t tid;
+  // userspace pid, kernel tgid
   pid_t pid;
-  pid_t tgid;
+  uid_t uid;
+  gid_t gid;
   u64 cgroup_id;
```

Comment:
make sure this matches exactly w readme.

Reply:
Was there something else besides your last comment that I messed up?